src/test-utils/src/render-test.js

```diff
@@ -122,11 +124,14 @@ export default class RenderTest {
 
   // Node test driver (puppeteer) may not have had time to expose the function
   // if the test suite is short. If not available, wait a second and try again
-  _reportToTestDriver(firstTime = false) {
+  _reportToTestDriver() {
     if (window.renderTestComplete) {
-      window.renderTestComplete(JSON.stringify(this.passed));
+      const result = {success: this.passed, failedTest: this.failedTest || ''};
```

Comment:
understood we want to report the failed the test, but we won't stop on the failed test to give developer a chance to compare or save the new image.

Reply:
This line is intended to do that, let me test mode: https://github.com/uber/deck.gl/pull/1520/files#diff-e2d6c6241b75fd4782a84292b157689eR115

Reply:
I see, the browser script stops, but the test harness still kills the browser. Will fix.

Reply:
Fixed